WinUIGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -46,13 +55,22 @@
                     ValueChanged="ProgressValue_ValueChanged"
                     VerticalAlignment="Center"/>
             </StackPanel>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <ComboBox x:Name="BackgroundComboBox2" SelectionChanged="Background_SelectionChanged2"  Header="Background color" PlaceholderText="Pick a color" Width="200">
+                        <x:String>Transparent</x:String>
+                        <x:String>LightGray</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
             <local:ControlExample.Xaml>
                 <x:String>
-                    &lt;ProgressRing Width="60" Height="60" Value="$(DeterminateProgressValue)" IsIndeterminate="False"/&gt;
+                    &lt;ProgressRing Width="60" Height="60" Value="$(DeterminateProgressValue)" IsIndeterminate="False" $(Background)/&gt;
```

Comment:
I think a line break for this would be good so that we don't have vertical scrolling.

Reply:
Fixed! :)